h2o-algos/src/main/java/hex/coxph/CoxPHModel.java

```diff
@@ -32,7 +32,16 @@
     final String _strata_column = "__strata";
     public String[] _stratify_by;
 
-    public enum CoxPHTies { efron, breslow }
+    /**
+     * True if model scoring should not be performed during the computation.
+     * This setting is mainly to support tests scenarios with {@link #_single_node_mode} set
+     * to true.
+     * 
+     * If true, no {@link CoxPHOutput#_training_metrics} is not computed and it's null.
+     */
+    boolean _skip_scoring = false;
```

Comment:
do you plan to expose this via py/R?

Reply:
I don't think it's necessary right now. It' mainly intended for java unit tests.

Reply:
just out of curiosity, why is this needed for tests?